evenOccurrences/src/evenOccurrences.js

```diff
@@ -0,0 +1,46 @@
+/*
+ *  * Find the first item that occurs an even number of times in an array.
+ *   * Remember to handle multiple even-occurance items and return the first one.
+ *    * Return null if there are no even-occurance items.
+ *    */
+
+/*
+ *  * example usage:
+ *   * const onlyEven = evenOccurence([1, 7, 2, 4, 5, 1, 6, 8, 9, 6, 4, 1]);
+ *    * console.log(onlyEven); //  4
+ *    */
+
+const arr = [1, 7, 2, 4, 5, 1, 6, 8, 9, 6, 4, 1];
+
+// ATTEMPT # 2 
+
+const evenOccurence = (arr) => {
+	let newArr	= arr.sort();
```

Comment:
Watch your code formatting here with the spaces between the newArr variable and the '='